src/cljs/rems/actions/return_action.cljs

```diff
@@ -3,39 +3,28 @@
             [reagent.core :as r]
```

Comment:
I have the feeling that these single-comment actions like return, close, comment could be unified. But please, don't do it in this PR! :D

Reply:
Often it looks like so until user feedback makes them slightly different each and then you need to have that single "component" customizable. I like these now because they are pretty KISS UI code.